azure-pipelines.yml

```
@@ -449,12 +449,12 @@ jobs:
       echo "== Testing pip install on Python 3.9 =="
       docker run \
         --rm -v $PYPI_FILES:/mnt/pypi python:3.9 \
-        /bin/bash -c "ls /mnt/pypi && pip install -f /mnt/pypi -q azure-cli==$CLI_VERSION.* && az self-test && az --version && sleep 5"
+        /bin/bash -c "cd /mnt/pypi && ls && pip install -f ./ azure_cli-$CLI_VERSION*whl && az self-test && az --version && sleep 5"
```

If we force it to install the `whl` packages, there will be no need to care about `$CLI_VERSION`. I guess we may simply use `azure_cli*.whl` to install all local wheels:

```suggestion
        /bin/bash -c "cd /mnt/pypi && ls && pip install -f ./ azure_cli*.whl && az self-test && az --version && sleep 5"
```

This will include 

```
$ echo azure_cli*.whl
azure_cli-2.51.0.post20230815081512-py3-none-any.whl
azure_cli_core-2.51.0.post20230815081512-py3-none-any.whl
azure_cli_telemetry-1.1.0.post20230815081512-py3-none-any.whl
```

---

`pip install azure_cli*.whl` also works, but I want to make this test command close to the real scenario: `pip install azure-cli`.